### PR TITLE
add a callback register interface of Part

### PR DESCRIPTION
**Pull request by liuyu85cn**

### CODE REVIEW

File: src/kvstore/Part.h [hunk 1]
```diff
@@ -114,11 +114,22 @@ class Part : public raftex::RaftPart {
...
 
+public:
+    struct CallbackOptions {
+        GraphSpaceID spaceId;
+        PartitionID partId;
+        TermID term;
+    };
+
+    using OnElectedCallBack = std::function<void(const CallbackOptions& opt)>;
+    void registerOnElected(OnElectedCallBack cb);
```
Comment (bright-starry-sky) [inline src/kvstore/Part.h:125]: Where do we need to call this method?

Comment (liuyu85cn) [reply src/kvstore/Part.h:125]: toss will

Comment (bright-starry-sky) [reply src/kvstore/Part.h:125]: > toss will

got

**Review by bright-starry-sky — DISMISSED** [review]

**Review by critical27 — DISMISSED** [review]
LGTM

**Comment by critical27** [comment]
Plz resolve the conflicts~

**Review by critical27 — APPROVED** [review]

**Review by bright-starry-sky — APPROVED** [review]

**Pull request merged**